programming-languages.md

```diff
@@ -1,51 +1,107 @@
 ---
-title: Server-side programming languages
+title: Programming languages
 layout: recommendation
-expires: 2017-09-01
+expires: 2017-10-01
 ---
 
-All programming languages have their own strengths and weaknesses.
-This means that we want to use a range of programming languages in
-order to build services and supporting tools to meet users needs.
-This document does not apply to choosing 'off the shelf' software
-(open source or not).
+We want to use a range of programming languages at GDS because we think
+using the right tool for the job will give us the best chance of building
```

Comment:
I think 'right tool for the job' isn't a great framing given what this document is trying to do. We're explicitly acknowledging that we're going to use tools which are 'good enough for the job' and align with skills and other things going on in GDS.